core/src/main/scala/org/http4s/sbt/Http4sOrgPlugin.scala

```diff
@@ -41,39 +41,15 @@ object Http4sOrgPlugin extends AutoPlugin {
 
   val githubActionsSettings: Seq[Setting[_]] =
     Seq(
-      spiewakMainBranches := Seq("main"),
+      tlCiReleaseBranches := Seq("main"),
       githubWorkflowJavaVersions := List("8", "11", "17").map(JavaSpec.temurin(_)),
-      githubWorkflowEnv += ("JABBA_INDEX" -> "https://github.com/typelevel/jdk-index/raw/main/index.json"),
-      githubWorkflowBuild := Seq(
-        WorkflowStep
-          .Sbt(List("scalafmtCheckAll", "scalafmtSbtCheck"), name = Some("Check formatting")),
-        WorkflowStep.Sbt(List("headerCheckAll"), name = Some("Check headers")),
-        WorkflowStep.Sbt(List("test:compile"), name = Some("Compile")),
-        WorkflowStep.Sbt(List("mimaReportBinaryIssues"), name = Some("Check binary compatibility")),
+      githubWorkflowBuild +=
         WorkflowStep.Sbt(
-          List("unusedCompileDependenciesTest"),
-          name = Some("Check unused compile dependencies")),
-        WorkflowStep.Sbt(List("test"), name = Some("Run tests")),
-        WorkflowStep.Sbt(List("doc"), name = Some("Build docs"))
-      ),
+          List("doc"),
+          name = Some("Build docs"),
+          cond = Some("matrix.ci == '' || matrix.ci == 'ciJVM'")),
       githubWorkflowBuildMatrixFailFast := Some(false),
-      githubWorkflowArtifactUpload := false,
```

Comment:
Any reason we don't like this?

Reply:
I remember it being an irritant, but don't remember why.  I'm happy to try without it.